<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Carrier File:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="carrierEdit"/>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QPushButton" name="carrierButton">
      <property name="text">
       <string>Select Carrier</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Secret File:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="secretEdit"/>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QPushButton" name="secretButton">
      <property name="text">
       <string>Select Secret</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Carrier File</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Secret File</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Output File</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="carrierImage">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QGraphicsView" name="graphicsView_3"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Console Output:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="encodeButton">
      <property name="text">
       <string>Encode Image</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="decodeButton">
      <property name="text">
       <string>Decode</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLabel" name="secretImage">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
